app/views/tasks/index.html.erb

```diff
@@ -0,0 +1,31 @@
+<section id="all_tasks">
+<h2>Do these things:</h2>
+<ol>
+  <% @tasks.each do |task| %>
+    <section id="list-tasks">
+    <% if task.completed == false %>
+      <li><strong><%= link_to "Task: #{task.name}", task_path(task.id) %></strong></li>
+    <% elsif task.completed == true %>
+      <li id="crossed-off"><%= link_to "Task: #{task.name}", task_path(task.id) %></li>
```

Comment:
Should this `li` have the `id` of `"crossed-off"`? Valid HTML says that IDs only apply to one element, where this ID is being applied to every `li` in this list.